pkg/bindinfo/handle.go

```diff
@@ -129,6 +129,9 @@ func (h *BindHandle) Update(fullLoad bool) (err error) {
 		timeCondition = fmt.Sprintf("WHERE update_time>'%s'", lastUpdateTime.String())
 	}
 
+	h.sctx.Lock()
+	defer h.sctx.Unlock()
+
 	// No need to acquire the session context lock for ExecRestrictedSQL, it
 	// uses another background session.
```

Comment:
Should we remove this comment?

Reply:
Will it affect the performance of high concurrent queries?

Reply:
@tangenta I removed the stale comment.

@fixdb No, this lock is only used by background jobs related to bindings. Normal queries won't be affected by this lock.